elasticdl/python/master/master.py

```diff
@@ -127,6 +144,18 @@ def __init__(self, args):
         self._should_stop = False
         self._exit_code = 0
 
+    def _set_completed_steps_by_checkpoint(self, checkpoint_dir_for_init):
```

Comment:
When will we call this method?

Reply:
We should call this method using `args.checkpoint_dir_for_init` after initializing callbacks. I have added the code to call it in the next commit.